### PR TITLE
Add Code4PuertoRico documentation

### LINKED ISSUE #1: Add CONTRIBUTING.md
Adding a this will help contributors know exactly how to interact with your repo and what to expect.

Creo que SuministrosPR tiene un [buen ejemplo](https://github.com/Code4PuertoRico/suministrospr/blob/master/CONTRIBUTING.md).

### LINKED ISSUE #3: Añadir codigo de conducta
Todo proyecto en Code 4 Puerto Rico debería tener un codigo de conducta que sea compatible con [a de Code 4 Puerto Rico](https://github.com/Code4PuertoRico/codeofconduct).

Puede ser tan sencillo como añadir un archivo CODE_OF_CONDUCT.md y apuntar al codigo de conducta de Code 4 PR.

### PR DESCRIPTION
**Pull request by rnegron**
## Details

Since this repository is now under the Code4PuertoRico organization, changes must be made and additional documents must be added in order to meet the community standards. 

Closes #3 and closes #1

### CODE REVIEW

**Comment by rnegron** [comment]
Ready to merge this, if it looks okay to you @froi !

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+## 💻 Code contributions
+
+### Finding a task (issue)
+
+* You can [search through the issues](https://github.com/Code4PuertoRico/municipios-api/issues) and see if there are any which you would like to work on.
+
+### Picking a task
+- If the issue does not yet have a contributor assigned to it and no one has commented that they would like to work on it, feel free to comment on the issue yourself, letting others know that you intend to work on it.
+
+
```
Comment (froi) [inline CONTRIBUTING.md]:
```suggestion
```

Comment (froi) [reply CONTRIBUTING.md]: Esta linea la podemos remover.

**Review by froi — APPROVED** [review]
Awesome.

Left you a nit. Up to you if you want to apply it or not. 🚢

**Pull request merged**